### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected end of JSON input

### DISCUSSION THREAD

**Discussion by mkwsra** [livewire/livewire#3268] [7. Bugs (archived)] [answered]
For some reason and out of the sudden this version `v2.5.2` of Livewire is producing this error, and I couldn't understand why, so if you can help that would be great

![image](https://user-images.githubusercontent.com/1743919/124914913-9388d100-dff9-11eb-985a-9a511bcf8305.png)

`index.js:29` content:
```javascript
 const initialData = JSON.parse(this.el.getAttribute('wire:initial-data'))
 this.el.removeAttribute('wire:initial-data')
```

Am I doing something wrong? because it was working just fine before

**Comment by mkwsra** [comment]
I just saw it's a duplicate of this #3252 sorry for opening a new one, however, this is every urgent 🙏

**Comment by mkwsra** [comment]
Downgrading to `v2.5.1` solved this issue

**Comment by joshhanley** [comment, marked as answer]
@mkwsra a fix has been published. Please upgrade to 2.5.3 🙂

**Reply by mkwsra** [reply]
Thanks a lot, I can see that this comment has been published for multiple threats, is it a bug from Github? or did you really do so? 😅

**Reply by joshhanley** [reply]
No, I commented on them all 🙂 each thread was on the same thing

**Reply by mkwsra** [reply]
Cool, thank you so much 🙌

**Reply by mkwsra** [reply]
Honestly, v2.5.3 didn't solve my problem I downgraded to v2.5.1 to solve this

**Reply by joshhanley** [reply]
@mkwsra another fix has been tagged for a different issue causing this error, please upgrade to Livewire 2.5.5 and let us know if you are still having any issues 🙂

**Comment by lislmaka** [comment]
hi,

the problem still exists. at least for me)
i am using laravel 8.12 / livewire 2.5.3 / vue 2.6.12

I was doing an composer update today and got this error. Everything worked fine yesterday

I've already tried everything and don't even know what else to try.

The page that generates the error contains a bit of the code for the vuejs. But vuejs code works well and without errors
On other pages where there is only livewire , there is no problem

what am I doing wrong ?

![error](https://user-images.githubusercontent.com/70829116/125101253-b216c700-e0e2-11eb-8235-4355f5b03a09.jpg)

**Reply by joshhanley** [reply]
@lislmaka can you confirm, is your `APP_DEBUG` set to true or false in you ENV file?

**Reply by lislmaka** [reply]
@joshhanley, hi
it doesn't matter for me
I have an error in both cases  :(

**Reply by joshhanley** [reply]
@lislmaka although it is the same error message, I believe you are running into a different issue related to Livewire and Vue. Can I suggest you open up your own bug report discussion?

**Reply by lislmaka** [reply]
@joshhanley ok, thanks, i will do it
